config/database.yml

```diff
@@ -15,7 +15,7 @@
 # gem 'pg'
 #
 default: &default
-  adapter: postgresql
+  adapter: postgis
```

Comment:
ℹ️ If you don't switch the adapter to postgis, it'll complain regularly about not knowing what postgis column types are.